util/lookupd/lookupd.go

```diff
@@ -243,9 +243,11 @@ func GetLookupdTopicProducers(topic string, lookupdHTTPAddrs []string) ([]string
 // GetNSQDTopics returns a []string containing all the topics
 // produced by the given nsqd
 func GetNSQDTopics(nsqdHTTPAddrs []string) ([]string, error) {
-	topics := make([]string, 0)
-	var lock sync.Mutex
-	var wg sync.WaitGroup
+	var (
```

Comment:
It's just the `make([]string, 0)`. We could just leave as

```
var topics []string
var lock sync.Mutex
var wg sync.WaitGroup
```

I changed to the var block because I prefer it, but I'm happy to leave the repeated `var` declarations if you like.